queries.sql

```diff
@@ -7,4 +7,21 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+DELETE from animals;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg <0;
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, AVG(escape_attempts) as avg_escape_attempts
+FROM animals
+GROUP BY neutered
+ORDER BY avg_escape_attempts DESC;
+SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight
+FROM animals
+GROUP BY species;
+SELECT species, AVG(escape_attempts) AS avg_escape_attempts
+FROM animals
+WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
+GROUP BY species;
```

Comment:
- [ ] **Good job so far 👏 👏** However, as we are required to add all queries to your queries.sql file, there are missing queries to be added to complete all the requirements  
   - **Please make sure also to demonstrate the starting and end of all transactions as suggested in the project required requirements**🙏

Reply:
@TuyishimireEric Do you mean all transactions including rollbacks and commits that I have used in the terminal?